### ISSUE TITLE
Training settings about SURREAL

### ISSUE THREAD

**Issue by zhangyujie9999** [yifita/deep_cage#9]
Now I get the training data of 3D-coded, and make a training experiment with 1000 data. But I got a bad result. Could you tell me how to set the parameters related to SURREAL?

In the process of training, I also found that in dataset.py, the values of source_normals and target_normals were set to none, which led to the two lines of code could not run.

`data["source_normals"] = torch.cat([_source_normals, _target_normals, _source_normals], dim=0).contiguous()`
`data["target_normals"] = torch.cat([_target_normals, _source_normals, _source_normals], dim=0).contiguous()`

I think if the value passed in is none, then these two lines of code will not work, so I annotated them. What negative impact will this have on the training process?

Looking forward to your help, thx!

**Comment by yifita** [comment]
Hi @zhangyujie9999 , thanks for trying out our code. Did you check out the code I recently uploaded? See the new Readme.md page for instructions for SURREAL. 
https://github.com/yifita/deep_cage#humanoid-deformations
